### PR TITLE
Modify rule S2318: LaYC format

### PR DESCRIPTION
**Pull request by joke1196**
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [ ] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

File: rules/S2318/python/rule.adoc [hunk 1]
```diff
@@ -1,25 +1,37 @@
+This rule raises an issue when the inequality operator `<>` is used.
+
 == Why is this an issue?
 
-The forms ``++<>++`` and ``++!=++`` are equivalent. But in Python 2.7.3 the ``++<>++`` form is considered obsolete.
+The operators ``++<>++`` and ``++!=++`` are equivalent, but since Python 2.7 the ``++<>++`` form is considered obsolete and is removed in newer version of Python.
+
```
Comment (maksim-grebeniuk-sonarsource) [inline rules/S2318/python/rule.adoc:6]: Should 'How to fix it' header be here?

**Review by maksim-grebeniuk-sonarsource — CHANGES_REQUESTED** [review]
Looks good, just left formatting question comment

File: rules/S2318/python/rule.adoc (hunk 1 above)
Comment (joke1196) [reply rules/S2318/python/rule.adoc:6]: No so if the rule is quite short we do not need to split it in two tabs. I think it makes sense in this one to keep everything in the main tab as it is a one-liner. You can check [here](https://github.com/SonarSource/rspec/blob/master/docs/description.adoc) to see which sections are optional or not.

File: rules/S2318/python/rule.adoc [hunk 2]
```diff
@@ -1,25 +1,37 @@
+This rule raises an issue when the inequality operator `<>` is used.
+
 == Why is this an issue?
 
-The forms ``++<>++`` and ``++!=++`` are equivalent. But in Python 2.7.3 the ``++<>++`` form is considered obsolete.
+The operators ``++<>++`` and ``++!=++`` are equivalent, but since Python 2.7 the ``++<>++`` form is considered obsolete and is removed in newer version of Python.
```
Comment (guillaume-dequenne-sonarsource) [inline rules/S2318/python/rule.adoc:5]: When has it been removed?
If it's been removed in Python 3, I'd just say "and has been removed in Python 3."
That way we don't imply this is a super recent development 😄

**Review by guillaume-dequenne-sonarsource — APPROVED** [review]
LGTM, minor comment, feel free to ignore.

**Review by maksim-grebeniuk-sonarsource — APPROVED** [review]
LGTM

File: rules/S2318/python/rule.adoc [hunk 3]
```diff
@@ -1,25 +1,37 @@
+This rule raises an issue when the inequality operator `<>` is used.
+
 == Why is this an issue?
 
-The forms ``++<>++`` and ``++!=++`` are equivalent. But in Python 2.7.3 the ``++<>++`` form is considered obsolete.
+The operators ``++<>++`` and ``++!=++`` are equivalent, but since Python 2.7 the ``++<>++`` form is considered obsolete and has been removed in Python 3.
```
Comment (jean-jimbo-sonarsource) [inline rules/S2318/python/rule.adoc:5]: We can phrase this better:

"The <> and != operators are equivalent. However, the <> operator is considered obsolete in Python 2.7 and has been removed from Python 3. Therefore, it is recommended to use != instead."

**Review by jean-jimbo-sonarsource — APPROVED** [review]
Left a rephrase comment

**Review by jean-jimbo-sonarsource — APPROVED** [review]

**Pull request merged**